_projects/2019/sunpy/helioviewer_draft.md

```diff
@@ -0,0 +1,59 @@
+---
+name: HelioViewer Python API Wrapper
+desc: Creating a full Python wrapper for https://api.helioviewer.org/docs/v2/.
+# add a description of your project
+requirements:
+ - Familiarity with Python
+ - Some familiarity with Unirest/HTTP requests (ideally)
+difficulty: medium
+issues:
+ - https://github.com/sunpy/sunpy/issues/2860
+ - https://github.com/sunpy/sunpy/issues/2762
+ - https://github.com/sunpy/sunpy/issues/2878
+mentors:
+ - FIND SOMEONE MAYBE
```

Comment:
I would be ok to mentor this one, @wafels would the helioviewer developer be able to help on this one?